### PR TITLE
Fix default page.

### LINKED ISSUE #162: Incorrect navigation after login
steps:
1) log in app;
2)log out
3) again log in 
check result
actual:
image

expected: home

### PR DESCRIPTION
**Pull request by Duracell19**
closed #162

### CODE REVIEW

File: ecommers/lib/ui/pages/shell_page.dart [hunk 1]
```diff
@@ -39,15 +39,14 @@ class ShellPage extends StatelessWidget {
...
-                  MorePage()
-                ]),
+            child:
+                IndexedStack(index: model.selectedItemIndex, children: <Widget>[
+              HomePage(),
+              SearchPage(),
+              CartPage(),
+              ProfilePage(),
+              MorePage(tabChanged: model.onTappedItem)
+            ]),
```
Comment (qkul) [inline ecommers/lib/ui/pages/shell_page.dart]: please add ","  after children "Widget []" to improve code readability

**Comment by fryette** [comment]
the issue here is that all pages after logout should be destroyed/ why pages are not destroyed?

File: ecommers/lib/ui/pages/more_page.dart [hunk 2]
```diff
@@ -97,6 +100,8 @@ class MorePage extends StatelessWidget {
   }
 
   Future logOutPressHandler() async {
+    tabChanged(0);
```
Comment (fryette) [inline ecommers/lib/ui/pages/more_page.dart:109]: all providers should be destroyed after logout.

Comment (fryette) [reply ecommers/lib/ui/pages/more_page.dart:109]: we don;t need to do this

**Review by fryette — CHANGES_REQUESTED** [review]

**Pull request closed without merging**